app/javascript/components/projects/organizations/ApplicationView.jsx

```diff
@@ -15,12 +15,29 @@ class ApplicationView extends React.Component {
         .getAttribute("content")
     };
     this.handleAccept = this.handleAccept.bind(this);
+    this.handleInterview = this.handleInterview.bind(this);
     this.handleReject = this.handleReject.bind(this);
   }
 
   handleAccept = e => {
     e.preventDefault();
+    console.log(error.response.data.message)
+    axios
+      .post(`/api/applications/${this.props.application.id}/decide`, {
+        decision: 'accepted'
+      })
+      .then(function(response) {
+        console.log(response);
+        window.location.reload();
+      })
+      .catch(function(error) {
+        console.log(error);
+      });
+  };
 
+  handleInterview = e => {
+    e.preventDefault();
+    console.log(error.response.data.message)
```

Comment:
Remove this

Reply:
this isnt my component

Reply:
oh wtf ur right oops. don't know why it's in the diff. could u remove it anyways? this could potentially break the view i think.

Reply:
Actually, i think we don't use `ApplicationView` anymore. Can you remove the entire file?